src/panels/lovelace/cards/hui-todo-list-card.ts

```diff
@@ -209,38 +225,42 @@ export class HuiTodoListCard extends LitElement implements LovelaceCard {
                   @click=${this._addItem}
                 >
                 </ha-icon-button>
-                <ha-textfield
-                  class="addBox"
-                  .placeholder=${this.hass!.localize(
-                    "ui.panel.lovelace.cards.todo-list.add_item"
-                  )}
-                  @keydown=${this._addKeyPress}
-                  .disabled=${unavailable}
-                ></ha-textfield>
               `
             : nothing}
+        </div>
+        <div class="header">
+          <span>
+            ${this.hass!.localize(
+              "ui.panel.lovelace.cards.todo-list.unchecked_items"
+            )}
+          </span>
           ${this.todoListSupportsFeature(TodoListEntityFeature.MOVE_TODO_ITEM)
-            ? html`
+            ? html`<ha-button-menu>
                 <ha-icon-button
-                  class="reorderButton"
-                  .path=${mdiSort}
-                  .title=${this.hass!.localize(
+                  slot="trigger"
+                  .path=${mdiDotsVertical}
+                ></ha-icon-button>
+                <ha-list-item @click=${this._toggleReorder} graphic="icon">
+                  ${this.hass!.localize(
                     "ui.panel.lovelace.cards.todo-list.reorder_items"
                   )}
-                  @click=${this._toggleReorder}
-                  .disabled=${unavailable}
-                >
-                </ha-icon-button>
-              `
+                  <ha-svg-icon
+                    slot="graphic"
+                    .path=${mdiSort}
+                    .disabled=${unavailable}
+                  >
+                  </ha-svg-icon>
+                </ha-list-item>
+              </ha-button-menu>`
             : nothing}
```

Comment:
Should we hide unchecked header if there is unchecked item?

![CleanShot 2023-12-21 at 16 21 43](https://github.com/home-assistant/frontend/assets/5878303/3a3a8110-7c62-445f-9e4a-5f96409970d2)

Reply:
not sure we should hide or just show a message underneath that `There are not items to do!`